src/main/java/org/spongepowered/api/text/format/TextColor.java

```diff
@@ -50,7 +50,9 @@
      *
      * @return the RGB background color of this text color
      */
-    Color getBackgroundColor();
+    default Color getBackgroundColor() {
+        return Color.of(getColor().toVector3i().div(4));
```

Comment:
Don't default implement this.

Reply:
I understand it's basic, but it is a valid property that every `TextColor` has. Are you suggesting to revert it to a interface method or remove it entirely?

Reply:
Just make it a regular interface method, the only things that should be default implemented is trivial calls to overloaded methods.